src/helix-ui/elements/HXFileTileElement.less

```diff
@@ -1,8 +1,7 @@
 @import "./HXElement";
 @import "base/links";
-@import "elements/hx-progress";
-@import "components/progress";
 @import (reference) "components/buttons";
+@import (reference) "components/progress";
```

Comment:
This shouldn't be `(reference)` because we need the progress bar CSS to be included in Shadow CSS.